nupic/research/frameworks/greedy_infomax/utils/loss_utils.py

```diff
@@ -0,0 +1,91 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2021, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+#
+# This work was based on the original Greedy InfoMax codebase from Sindy Lowe:
+# https://github.com/loeweX/Greedy_InfoMax
+# The Greedy InfoMax paper can be found here:
+# https://arxiv.org/abs/1905.11786
+# ----------------------------------------------------------------------
+
+import torch
+import torch.nn.functional as F
+
+
+def multiple_cross_entropy(data_lists, targets, reduction="mean"):
+    log_f_module_list, true_f_module_list = data_lists
+    device = log_f_module_list[0][0].device
+    total_loss = torch.tensor(0.0, requires_grad=True, device=device)
```

Comment:
I don't think 'multiple cross entropy' is widely-used term, so please add documentation for this function.

Reply:
Done